apps/client/src/pages/CheckerPage.js

```diff
@@ -0,0 +1,231 @@
+import React, { useContext } from "react";
+import { Helmet } from "react-helmet";
+import { Redirect } from "react-router-dom";
+
+import Conclusion from "../components/Conclusion";
+import DebugDecisionTable from "../components/DebugDecisionTable";
+import Layout from "../components/Layouts/DefaultLayout";
+import LocationInput from "../components/Location/LocationInput";
+import LocationResult from "../components/Location/LocationResult";
+import PrintDetails from "../components/PrintDetails";
+import Questions from "../components/Questions";
+import {
+  StepByStepItem,
+  StepByStepNavigation,
+} from "../components/StepByStepNavigation";
+import { SessionContext } from "../context";
+import withChecker from "../hoc/withChecker";
+import { geturl, routes } from "../routes";
+
+const CheckerPage = ({ checker, topic, resetChecker }) => {
```

Comment:
I guess this would be the place to handle logic for setting the active / finished states of the step by step items.